portfolio/src/main/webapp/index.html

```diff
@@ -27,7 +27,7 @@
       <div id = "intro">
           <div class = "content">
               <div class = "column" style="padding-top:30px;">
```

Comment:
Consider defining style in one spot at the top or maybe in a different file. This will make your html much more readable